src/renderer/state/modal.ts

```diff
@@ -0,0 +1,49 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import type { RootState } from './store';
+
+export interface ModalScreen {
+  route: string | undefined;
+  type: 'alert' | 'modal' | undefined;
+}
+
+export interface ModalConfig {
+  preferences: object;
```

Comment:
Should these be optional params? thinking modalConfig for nodeSettings wouldn't want to set the other values

Reply:
Oohh good find, this is some old code and we have ModalConfig interface in modalUtils so this isn't necessary. will delete